src/parser.c

```diff
@@ -1886,6 +1886,7 @@ static bool ts_lex(TSLexer *lexer, TSStateId state) {
       if (lookahead == '~') ADVANCE(89);
       if (lookahead == '\t' ||
           lookahead == '\n' ||
+          lookahead == '' ||
```

Comment:
I'm worried that this character literal won't be accepted by all C compilers. Should `tree-sitter` be generating some escape sequence for form feed characters?

Reply:
Tree-sitter should probably output this as `'\f'`. I guess if both Travis and Appveyor succeed, it's probably fine though.

Reply:
it was my answer :)
Anyway it's probably better to escape it when generating the C code

Reply:
Ok, I opened an issue about this https://github.com/tree-sitter/tree-sitter/issues/487. I think it's ok for now though, since CI appears to pass.